include/loads_bcs/acceleration_constraint.h

```diff
@@ -0,0 +1,56 @@
+#ifndef MPM_ACCELERATION_CONSTRAINT_H_
+#define MPM_ACCELERATION_CONSTRAINT_H_
+
+//! Alias for JSON
+#include "json.hpp"
+using Json = nlohmann::json;
+
+#include "function_base.h"
+
+namespace mpm {
+
+//! AccelerationConstraint class to store acceleration constraint on a set
+//! \brief AccelerationConstraint class to store a constraint on a set
+//! \details AccelerationConstraint stores the constraint as a static value
+class AccelerationConstraint {
```

Comment:
Is there a general constraint class from which this could be derived?

Reply:
Currently, no. But I do agree that we should implement a general constraint from which this one and others (`VelocityConstraint`, `FrictionConstraint`, `Traction`) should be derived. They all have a setid_, dir_ and their main value (acceleration_, velocity_, traction_, or friction_). They only differ on the way these main values are named and if they can or cannot be time-dependent -- i.e. if they can have a pointer to a math function. Currently, only traction and acceleration constraints can be time-dependent.